mechaclient.py

```diff
@@ -139,3 +141,32 @@ def api_handler(self) -> object:
         Mecha's API connection
         """
         return self._api_handler
+
+    @property
+    def board(self) -> RatBoard:
+        """
+        Mecha's RatBoard Object
+
+        """
+        if not self._rat_board:
+            self._rat_board = RatBoard()  # Create Rat Board Object
+        return self._rat_board
+
+    @board.setter
+    def board(self, value):
+        """
+        Mecha's RatBoard Setter
+        """
+        if not isinstance(value, RatBoard):
+            raise TypeError("board property must be of type RatBoard.")
+
+        log.warning("Board Setter invoked!")
+        self._rat_board = value
+
+    @board.deleter
+    def board(self):
+        """
+        Mecha's Ratboard Deleter
```

Comment:
[nitpick] drop `Mecha's` in the property docstrings, IMHO its redundant since it is an instance method of the `MechaClient` as it were...